.travis.yml

```diff
@@ -14,6 +14,7 @@ script:
   - yarn run pretty-check
   - solium -d contracts/
   - yarn run coverage && cat ./coverage/lcov.info | coveralls
+  - yarn run test ./test/stablex/stablex_large_example.js
```

Comment:
for this you need an instance of ganache to run

Reply:
Right... dang.